data/application_manifest.xml

```diff
@@ -25,4 +20,9 @@
             </requestedPrivileges>
         </security>
     </trustInfo>
+    <asmv3:application>
+        <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">
+            <activeCodePage>UTF-8</activeCodePage>
+        </asmv3:windowsSettings>
+    </asmv3:application>
```

Comment:
Why was this moved?

Reply:
One of the linkers or resource compiler (don't remember which one on what platform) was reordering it in the final executable. So I moved the same way it had appear in that one.